### DISCUSSION TITLE
Handling composite types

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#4259] [Q&A]
I recently tried to implement a composite type but found, as the docs state, that it is not supported. In my postgres db, I created a type called side which is composed of an Integer type column and a text type column. I then have a table with a column called sides which is of type side[]. However, when I introspected my db, it gave me this:

```
// This type is currently not supported.
// sides _side[]
```
Is there any way to implement composite types or what would be the best way to go about handling this. I would really prefer to be able to use this composite type as keeping those two values together will be quite useful but I do not want to create a separate table for it.

**Comment by ryands17** [comment]
Hey @jejacobsendev 👋 
Composite types are not supported by Prisma yet, I would suggest creating a feature request for that [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
